app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(name=person["name"], age=person["age"])
+                   for person in people
+                   ]
```

Comment:
fix formating here
ur openning bracket should have blank line next to it if ur closing bracket is starting from new line 
f.e. u can make like this
`person_list = [`
`       <some var>,`
`       <some var>`
`]`
OR
`person_list = [<some var>,`
`               <some var>]`